test/authorization.spec.js

```diff
@@ -47,6 +47,61 @@ describe('authorization', function () {
         });
     });
 
+    describe('when authorizationStrategy returns custom JSONAPI_Error', function () {
```

Comment:
only 1 issue with yuor tests - they pretty much excersize 403's, and 403 was the issue earlier
so if there was an issue, you might not catch it
may make sense to throw an error code that isn't really used in the app, like 499